tcp/server/config.go

```diff
@@ -0,0 +1,67 @@
+package server
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/plgd-dev/go-coap/v3/message"
+	"github.com/plgd-dev/go-coap/v3/message/codes"
+	"github.com/plgd-dev/go-coap/v3/message/pool"
+	"github.com/plgd-dev/go-coap/v3/net/monitor/inactivity"
+	"github.com/plgd-dev/go-coap/v3/net/responsewriter"
+	"github.com/plgd-dev/go-coap/v3/options/config"
+	"github.com/plgd-dev/go-coap/v3/tcp/client"
+)
+
+// A ServerOption sets options such as credentials, codec and keepalive parameters, etc.
+type TCPServerOption interface {
+	TCPServerApply(*Config)
+}
+
+// The HandlerFunc type is an adapter to allow the use of
+// ordinary functions as COAP handlers.
+type HandlerFunc = func(*responsewriter.ResponseWriter[*client.ClientConn], *pool.Message)
+
+type ErrorFunc = func(error)
+
+type GoPoolFunc = func(func()) error
+
+// OnNewClientConnFunc is the callback for new connections.
+type OnNewClientConnFunc = func(cc *client.ClientConn)
+
+var DefaultConfig = func() Config {
+	opts := Config{
+		Common: config.DefaultCommon(),
+		CreateInactivityMonitor: func() client.InactivityMonitor {
```

Comment:
Is this deliberate? Previously, the default was a nil monitor.

tcp/server.go:
```go
createInactivityMonitor: func() inactivity.Monitor {
    return inactivity.NewNilMonitor()
},
```

Reply:
Yes, it is intentional. TCP Server want's to watch if clients are reachable by default.